src/site/content/en/blog/badging-api/index.md

```diff
@@ -159,9 +154,57 @@ Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
 then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
+`setAppBadge(unreadCount)` and let the user agent deal with displaying it
 accordingly.
 
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
+on the install state. Just call the API when it exists. If it works,
+it works. If not, it simply doesn't.
+
+{% Aside %}
+  Once the origin trial is completed, the API should switch to
+  `navigator.setAppBadge()`.
```

Comment:
I suspect we don't want to say this. While we're pretty confident (but not certain) that this is what we'll move to I don't think we want to encourage developers to set their app up to use this.

Part of the point of the origin trial is that things break (which is why we had to name the API `xxxExperimentalAppBadge` in the first place).

Reply:
I agree. The practical subtext of this paragraph is "you can just detect `setAppBadge` or `setExperimentalAppBadge` and call whichever is available". We don't want people predicting what the final API is going to be, because otherwise they won't have to do any work to change over to it when we launch.

So remove this para.

Reply:
Removed.